samples/boards/nrf52/mesh/onoff_level_app/src/main.c

```diff
@@ -0,0 +1,38 @@
+#include "common.h"
+
+void main(void)
+{
+	int err, rc = 0;
+
+	rc = nvs_init(&fs, FLASH_DEV_NAME, STORAGE_MAGIC);
+
```

Comment:
redundant empty line (generally, no empty line between assignment to variable and testing for the variable's value)